### ISSUE TITLE
Bug for screen reader compatibility on Windows

### ISSUE THREAD

**Issue by florentleveque** [warpdotdev/Warp#7242]
### Pre-submit Checks

- [x] I have [searched Warp bugs](https://github.com/warpdotdev/warp/issues?q=is%3Aissue+label%3ABugs) and there are no duplicates
- [x] I have [searched Warp known issues page](https://docs.warp.dev/help/known-issues) and my issue is not there
- [ ] I have an issue with AI and have included the debugging ID (Optional, but helps expedite the AI quality fix). [Debugging ID intructions](https://docs.warp.dev/agents/ai-faqs#gathering-ai-debugging-id)
- [ ] I have technical issue and have included the logs (optional, but helps expedite the bug fix). [Log intructions](https://docs.warp.dev/support-and-billing/sending-us-feedback#gathering-warp-logs)

### Describe the bug

I'm blind and use the NVDA screen reader to use Windows.

When I launch Warp on Windows, I can't do anyThing the screen reader does not announce anything in braille or vocally.

I can just have the output of a command in the terminal vocally but that's all.

I have tested it with the Windows narator but I have the samme issue.

### To reproduce

* [Download nvda screen reader](https://www.nvaccess.org/download/)
* Install it
* launch Warp terminal
* Type on the keyboard and listen the vocal informations given by NVDA screen reader.
* To reproduce the braille issues you must to have a braille terminal to link to your computer, if it is not possible for you, I can test your updates 
* To have more informations about NVDA, please contact NVAccess whitch develop NVDA.

### Expected behavior

For the momment Warp is not accessible, and the in the warp first view for example, NVDA announces northing at all.

In a Terminal, you just have the output terminal command  vocally but not in braille.

If I type in the terminal, Nvda announces vocally (not in braille) just the chars I type, but the touch backspace for example have not the nvda output.

I can not  test other thing for the momment.

### Operating system (OS)

Windows

### Operating system and version

11

### Shell Version

WSL

### Current Warp version

The last today

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Recent working Warp date

never

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this an issue only in Warp?

Yes, I confirmed that this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None
